client/admin/media/index.jsx

```diff
@@ -0,0 +1,142 @@
+import React, { useRef } from 'react';
+import Button from '@material-ui/core/Button';
+import { List, useListController, useMutation } from 'react-admin';
+import MuiGridList from '@material-ui/core/GridList';
+import GridListTile from '@material-ui/core/GridListTile';
+import { makeStyles } from '@material-ui/core/styles';
+import AddIcon from '@material-ui/icons/Add';
+import { API_URL } from '../constants';
+import { useWidth } from '../hooks';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    padding: 4,
+  },
+  gridList: {
+    width: '100%',
+    margin: '0px !important',
+  },
+  tileBar: {
+    background:
+      'linear-gradient(to top, rgba(0,0,0,0.8) 0%,rgba(0,0,0,0.4) 70%,rgba(0,0,0,0) 100%)',
+  },
+  placeholder: {
+    backgroundColor: theme.palette.grey[300],
+    height: '100%',
+  },
+  price: {
+    display: 'inline',
+    fontSize: '1em',
+  },
+  link: {
+    color: '#fff',
+  },
+  uploadBtn: {
+    color: '#3f51b5',
+    fontSize: '0.8125rem',
+  },
+  imageSelected: {
+    filter: `blur(1px)`,
+  },
+}));
+
+const getColsForWidth = width => {
+  if (width === 'xs') return 2;
+  if (width === 'sm') return 3;
+  if (width === 'md') return 4;
+  if (width === 'lg') return 5;
+  return 6;
+};
+
+const times = (nbChildren, fn) => Array.from({ length: nbChildren }, (_, key) => fn(key));
+
+const LoadingGridList = ({ nbItems = 10 }) => {
+  const width = useWidth();
+  const classes = useStyles();
+  return (
+    <div className={classes.root}>
+      <MuiGridList cellHeight={180} cols={getColsForWidth(width)} className={classes.gridList}>
```

Comment:
can you create a CELL_HEIGHT const? also why it is 180 instead of 200 as here - https://github.com/flatlify/flatlify-react-admin-example/pull/2/files#diff-c9c0fea1f62807e6555cdff050fdbd4fR109 ?